active/social-media.md

```diff
@@ -0,0 +1,72 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability by generating revenue through sponsor relationships/campaigns.
+
+The group opts to manage platforms that best achieve these goals - which may be a subset of where Django is or could be engaging.
+
+Delegated responsibilities:
+- The Chair can grant access to Django's official social media profiles and each member will have access.
+- Members can post content to Django's official social media profiles, in alignment with the working group's agreed goals, without a board review.
+- Members can like/repost/comment as Django, when it aligns with the working group's agreed goals, without a board review.
+- The working group can post content on djangoproject.com, when it aligns with the working group's agreed goals, without a board review.
+
+Actions to take back to the DSF Board for votes:
+- Amendments to the working group's goals.
+- Approval on content that is outside the agreed group's goals.
+- Proposals of social media campaigns for our sponsors.
+- Proposal to have a new social media profile on a platform.
+- Proposal to retire a social media profile from a platform.
+
+## Initial membership
+
+- Chair:
+- Co-Chair:
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair):
+- Other members:
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have demonstrable experience with social media platforms, and be registered as a Django Software Foundation Individual Member.
+
+### How do people who want to join sign up / volunteer / express interest?
+
+By sending an email to: `social-media-wg@djangoproject.com`
+
+This email should include:
+- why they want to join
+- what experience/skills they bring to the role
+- links to their social media profiles
+
+All members should have access to this email.
+
+### How will decisions on adding/removing members be handled?
+
+The working group will discuss and vote (50%+1) to approve new members.
+
+Members join the group for a 6-month term. At the end of this term, they need to opt into staying involve to keep being a member of the group.
+
+If any member wishes to leave the group before the end of their term, they can do so without a vote.
+
+Members can propose a vote on removing a member from the working group. This needs 50%+1 agreement.
+
+## Budget
+
+The working group will have no allocated budget.
+
+## Comms within the group
+
+- Fortnightly synchronous meeting
+- Collaborative documents when creating a content plan and/our campaign
+- Private channel on the Django discord server
```

Comment:
```suggestion
- Establish a private communication channel with logs.
```

I would just remove this and let the WG decide itself which communication platform to use :)